lib/measured/units/dimension.rb

```diff
@@ -0,0 +1,26 @@
+class Measured::Dimension < Measured::Measurable
```

Comment:
NIST Special Publication 1038: The International System of Units (SI) – Conversion Factors for General Use (http://www.nist.gov/pml/wmd/metric/upload/SP1038.pdf ) is a document I was double checking our conversion with. Looks good!

But they refer to dimension as length, for example, "the kilometer measures length". .